Pledge-Based Saturation Limit Under a Closed System/README.md

```diff
@@ -0,0 +1,252 @@
+  - a **closed system** is maintained, which means that the ecosystem total saturation limit is always equal to the total ADA in circulation regardless of the pledge status of the ecosystem, _i.e._, under-pledged, fully-pledged, or over-pledged. **TRULY OUROBOROS**.
+  - no single pool has a saturation limit that dwarfs other pools’ limits, encouraging decentralization.
+  - since the ecosystem total saturation limit is always equal to ADA in circulation, the ADA delegation per pool will be more or less even.
+* Optimal pledge decreases down to a minimum as the total number of pools increases but increases exponentially as the total number of pools decreases. This mechanism addresses Sybil attack:
+  - optimal pledge becomes expensive when the total number of pools decreases, thereby increasing the expense to conduct a Sybil attack.
+  - expense to conduct Sybil attack also increases as the total number of pools increases because although the optimal pledge decreases, optimal saturation limit decreases as well. Therefore, the attacker needs to setup a very large number of pools in order to obtain 51% of the total delegated ADA.
+  
+## **Mathematical Description**
+### _**Calculating Optimal Pledge**_<br>
+<br>
+  
+![equation1](https://drive.google.com/uc?export=view&id=1O_gmZ9LwpVV3xs2RVNqaXLh3um3zfFKs)
+
+where **_p<sub>opt</sub>_** is the optimal pledge, **φ** is the amount of ADA in circulation (currently 33B), **_k_** is the total number of pools, and **_n_** is a variable that is determined via consensus to warrant an economically viable optimal pledge (**_p<sub>opt</sub>_**). Therefore, it appears that **_n_** is a variable that the community can vote for adjustment to increase or decrease **_p<sub>opt</sub>_**. In effect, it can be used to control the total number of pools (**_k_**) so that at some value **_n_**, the ecosystem gradually settles to some value **_k_** which the community thinks is the optimal value. It is very important that **_k_** is tightly controlled by **_n_**, and **_k_** should only increase according to some parameters, for example, with increasing user demand. So that even if **_k_** is large, all pools are still minting blocks because of high user demand.
+  
+If we set **_n_** =15,
+
+**scenario 1** (very small number of pools):  
+* **_k_** = 500,
+* **_p<sub>opt</sub>_**  = ~ 774,763 ADA per pool
+
+**scenario 2** (current number of pools):
+* **_k_** = 3119 (epoch 321)
+* **_p<sub>opt</sub>_**  = 47,205 ADA per pool
+
+**scenario 3** (very large number of pools):  
+* **_k_** = infinity
+* **_p<sub>opt</sub>_**  = 27,440 ADA per pool  

```

Comment:
I think those results could be better presented in a table:

| $k$ | $p_{opt}$ | 
| --- | --- | 
| `500` | `774 763` |
| `3119`<sup>*</sup> | `47 205` | 
| $+\infty$ | `27 440` |

> (*) Current number of pools as per epoch 321.

Alternatively, could show a plot of equation for variable $k$ and various values of $n$.